src/common/utils.ts

```diff
@@ -114,3 +112,42 @@ export function stringToHslColor(str: string, saturation: number, lightness: num
   const hue = hash % 360;
   return `hsl(${hue}, ${saturation}%, ${lightness}%)`;
 }
+
+export function extractPhraseFromString(value: string) {
```

Comment:
Can we add some unit tests for these new functions? It would also help me better understand what the input -> output looks like.

Reply:
I added them for this method, but not the paste method as it seems to be not straightforward to emulate clipboard events in playwright